sdk/src/test/scala/io/horizen/account/wallet/AccountWalletTest.scala

```diff
@@ -209,8 +209,8 @@ class AccountWalletTest
   @Test
   def testGenerateSecretsOfDifferentDomainsIndependent(): Unit = {
```

Comment:
At line 262 the assert message is wrong (probably a copy-paste)

Reply:
The message is still wrong because it is not a schnorr key but a 25519 key

Reply:
Done